### PR TITLE
Add @cameron-martin to CODEOWNERS

### PR DESCRIPTION
**Pull request by jfirebaugh**
Per https://github.com/bazelbuild/vscode-bazel/issues/241#issuecomment-1900405902. Thank you for offering!

I think there's a second step that I don't have permission to do, which is to add @cameron-martin as a collaborator with write access. At least that's what GitHub is telling me:

Screenshot 2024-01-22 at 2 56 44 PM

### CODE REVIEW

**Comment by meteorcloudy** [comment]
I sent an invitation to @cameron-martin

**Review by meteorcloudy — APPROVED** [review]

**Comment by cameron-martin** [comment]
@coeuvre I think this needs a review from you too, since you're the only other code owner.

**Review by coeuvre — APPROVED** [review]

**Pull request merged**
